apps/src/metrics/statsigHelpers.js

```diff
@@ -13,15 +11,8 @@ export function getUserType() {
 }
 
 export function findOrCreateStableId() {
```

Comment:
*nit* This doesn't "Create" a stable ID anymore.

Reply:
Good call. Actually now I'm wondering if I _should_ have it create a stable ID in the case it doesn't get one passed to it from the backend 🤔

Reply:
Well, except that we wouldn't have anyplace to store it if we created one here. This would run on each page load and generate a new ID each time, which would not be very "stable". I think I'll just change the name and expect that it will be passed from the backend.